package.json

```diff
@@ -74,6 +74,8 @@
     "read-pkg": "^9.0.1",
     "release-please": "^17.1.3",
     "storybook": "^9.0.15",
+    "tailwind-merge": "^3.4.0",
+    "tailwind-variants": "^3.1.1",
```

Comment:
since these are imported in components should they be promoted from dev dependencies?

Reply:
I had to add this to make the storybook build work. I'm not sure why. They are already deps of the uds package.